crates/bevy_ecs/src/query/world_query.rs

```diff
@@ -14,11 +13,11 @@ use variadics_please::all_tuples;
 /// # Safety
 ///
 /// Implementor must ensure that
-/// [`update_component_access`], [`matches_component_set`], [`fetch`] and [`init_fetch`]
+/// [`update_component_access`], [`matches_component_set`], [`QueryData::fetch`] and [`init_fetch`]
```

Comment:
We probably want to mention `QueryFilter::filter_fetch` alongside `QueryData::fetch` here and below.